csg.js

```diff
@@ -6109,14 +6129,44 @@ for solid CAD anyway.
                 for (var ii = i + 1; ii < numsides; ii++) {
                     var side1 = this.sides[ii];
                     if (CAG.linesIntersect(side0.vertex0.pos, side0.vertex1.pos, side1.vertex0.pos, side1.vertex1.pos)) {
-                        if (debug) { OpenJsCad.log(side0); OpenJsCad.log(side1);}
+                        if (debug) { console.log("side "+i+": "+side0); console.log("side "+ii+": "+side1);}
```

Comment:
Good change ! 
even better to use es6 template strings like 
console.log(`side ${i} ${side0}`
console.log(`side ${ii} ${side1}`